content/en/docs/_index.md

```diff
@@ -28,9 +28,9 @@ Falco is deployed as a long-running daemon. You can install it as a [Debian](ins
 
 Falco is configured via (1) a [rules file](rules) that defines which behaviors and events to watch for and (2) a [general configuration file](config). Rules are expressed in a high-level, human-readable language. We've provided a sample rules file [`./rules/falco_rules.yaml`](https://github.com/falcosecurity/falco/blob/dev/rules/falco_rules.yaml) as a starting point—you can (and will likely want!) to adapt it to your environment.
 
-When developing rules, one helpful feature is Falco's ability to read trace files saved by Sysdig. This allows you to "record" the offending behavior once and replay it with Falco as many times as needed while tweaking your rules.
+When developing rules, one helpful feature is Falco's ability to read trace files saved by the `sysdig ` command line tool. This allows you to "record" the offending behavior once and replay it with Falco as many times as needed while tweaking your rules.
```

Comment:
```suggestion
When developing rules, one helpful feature is Falco's ability to read trace files saved by the `sysdig` command line tool. This allows you to "record" the offending behavior once and replay it with Falco as many times as needed while tweaking your rules.
```

Reply:
I would not say `files saved by the sysdig command line tool`, I would say `files saved in the scap format`

Reply:
Okay I fix-ed it